command/intake.py

```diff
@@ -0,0 +1,47 @@
+from robotpy_toolkit_7407.command import SubsystemCommand
+from subsystem import Intake
+
+class DeployIntake(SubsystemCommand[Intake]):
```

Comment:
Write a command that sets the intake at an angle (take a look at SetWrist from the Goober)